app/views/order_management/order_details/_warnings.html.haml

```diff
@@ -2,7 +2,8 @@
 - if notices[:warnings].any?
   .alert.alert-error
     - notices[:warnings].each do |warning|
-      = t("order_details.notices.#{warning}.alert")
+      -# TODO: Should be getting fixed in another PR
```

Comment:
That other PR already existed. Now with a merge/rebase off master, this isn't needed anymore, so I'm removing it.